src/main/java/seedu/duke/Parser.java

```diff
@@ -75,7 +75,7 @@ public void checkIllegalCharacter(String userInput) throws IllegalCharacterExcep
     }
 
     public void checkEmptyCommand(String parsedInput) throws InvalidCommandException {
-        if (parsedInput.isEmpty()) {
+        if (parsedInput.isBlank()) {
```

Comment:
this ensures that its invalid input if user just inputs whitespace for a title or other param